replicator/src/main/scala/com/evolutiongaming/kafka/journal/replicator/Batch.scala

```diff
@@ -14,7 +14,7 @@ private[journal] object Batch {
 
   def of(records: Nel[ActionRecord[Action]]): List[Batch] = {
 
-    def cut(appends: Appends, delete: Action.Delete) = {
+    def dropToBeDeletedAppends(appends: Appends, delete: Action.Delete): Boolean = {
```

Comment:
Sorry but I do not understand the name, might be docs will help? Actually _if only docs help_ then method name can stay as before